lib/sa.ts

```diff
@@ -16,6 +16,11 @@ export const navigateToSAList = async function (page: Page) {
   await expect(page.locator('h1', { hasText: 'Service Accounts' })).toHaveCount(1);
 };
 
+export const navigateToServiceAccounts = async function (page: Page) {
```

Comment:
Why do you need this new function instead of using the above one?

Reply:
It seems it is just a leftover

Reply:
We developed the same function in parallel I think. I leave yours

Reply:
Can we resolve this conversation @Frawless ?